app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -0,0 +1,116 @@
+import { Metadata } from 'next';
+import {
+  getStoryblokApi,
+  ISbStoriesParams,
+  StoryblokClient,
+} from '@storyblok/react/rsc';
+import StoryblokStory from '@storyblok/react/story';
+import { resolveRelations } from '@/utilities/resolveRelations';
+import { getPageMetadata } from '@/utilities/getPageMetadata';
+import { notFound } from 'next/navigation';
+
+const activeEnv = process.env.NODE_ENV || 'development';
+
+type PathsType = {
+  slug: string[];
+};
+
+type ParamsType = {
+  slug: string[];
+};
+
+// Control what happens when a dynamic segment is visited that was not generated with generateStaticParams.
+export const dynamicParams = false;
+
+/**
+ * Generate the list of stories to statically render.
+ */
+export async function generateStaticParams() {
+  // Client in the App router is tied to the front end initialization.
+  const storyblokApi: StoryblokClient = getStoryblokApi();
+  let sbParams: ISbStoriesParams = {
+    version: activeEnv === 'development' ? 'draft' : 'published',
+    cv: activeEnv === 'development' ? Date.now() : undefined,
+  };
+
+  // Use the `cdn/links` endpoint to get a list of all stories without all the extra data.
+  const response = await storyblokApi.getAll('cdn/links', sbParams);
+  const stories = response.filter((link) => link.is_folder === false);
```

Comment:
Only filter out folders now. We will render the home page with this route as well.

Reply:
Cool - this works well! I like how you consolidate the 2 pages file to support the "home" page also 😄 

This page which is set as root inside a folder still works 👍🏼 
https://deploy-preview-127--giving-campaign.netlify.app/initiatives